src/Adapter/OrderReturn/CommandHandler/DeleteProductFromOrderReturnHandler.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
There's no legacy here, so this should be placed inside the core.

Reply:
Moved